optimization/ht_functions.py

```diff
@@ -111,7 +104,6 @@ class Flow:
     dp = 0  # channel pressure drop
 
     # heat transfer attributes
-    dt = pc.T_centerline - pc.T_bulk  # temp. drop fuel -> coolant
     h_bar = 0  # average heat transfer coefficient
     Nu = 0  # Nusselt number
```

Comment:
Do we still need `Nu` here ?

Reply:
Same for `Re` above...

Reply:
I removed these. Going to wait on the PEP8 filters until @gonuke gives the final thumbs up. I don't want to pollute the changes.